package.json

```diff
@@ -13,7 +13,7 @@
     "db:console": "docker exec -it prescare-database psql -U postgres -d prescare",
     "db:reset": "npm run db:stop;sleep 1; npm run db:start; sleep 2; npm run db:create",
     "db:stop": "docker stop prescare-database",
-    "db:seed": "node seed.js"
+    "db:seed": "cat seed.sql | docker exec -i prescare-database psql -U postgres -d prescare"
```

Comment:
A gente havia comentado sobre mudar os seeds para usar queries Sequelize em vez de SQL puro. Isso acabou acontecendo?

Reply:
Sim, já foi feito.

Reply:
Tanto que se tu for ver, o código antigo rodava `node seed.js`